portfolio/src/main/java/com/google/sps/data/PortfolioComments.java

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.ArrayList;
+
+
+/**
+* Class containing list of Comments for Jalyn's Portfolio 
```

Comment:
class is not needed and class comment is to describe what a class is about for model class or what feature for util class.

Reply:
Piggyback off Tianxiao's comment, I think this class currently isn't needed unless there are specific methods that would be added not provided by the ArrayList